packages/mobile/src/transactions/TransactionsList.tsx

```diff
@@ -209,28 +209,7 @@ function mapInvite(tx: FeedItem): FeedItem {
   return tx
 }
 
-function getTransactions(data: UserTransactionsQuery | undefined) {
-  return data?.tokenTransactions?.edges.map((edge) => edge.node).filter(isPresent) ?? []
-}
-
 export class TransactionsList extends React.PureComponent<Props> {
```

Comment:
Nit: can this be refactored to be a functional component?

Reply:
Maybe but I'll avoid temptation to make this PR bigger :)